backend/src/ai_ticket_platform/services/infra/storage/azure.py

```
@@ -113,3 +113,53 @@ def put_presigned_url(
 			verify_exists=False,
 			**params,
 		)
+
+	def upload_blob(self, blob_name: str, content: Union[str, bytes], content_type: Optional[str] = None) -> str:
+		"""
+		Upload content (string or binary) directly to Azure Blob Storage using backend credentials.
+
+		Args:
+		    blob_name: Name/path of the blob
```

Ruff (0.14.7)

Indentation contains mixed spaces and tabs

(E101)